client/src/components/LibraryFolder/LibraryFolder.vue

```diff
@@ -153,6 +153,10 @@
                     <font-awesome-icon v-if="row.item.type === 'folder'" :icon="['far', 'folder']" title="Folder" />
                     <font-awesome-icon v-else-if="row.item.type === 'file'" title="Dataset" :icon="['far', 'file']" />
                 </template>
+                <template v-slot:cell(type)="row">
+                    <div v-if="row.item.type === 'folder'">{{ row.item.type }}</div>
+                    <div v-else-if="row.item.type === 'file'">{{ row.item.file_ext }}</div>
+                </template>
```

Comment:
Should we merged these 2 columns? For a folder it could be the icon, for a dataset its datatype.

Reply:
If we merge them, the folder's type would be empty

Reply:
Currently:
![image](https://user-images.githubusercontent.com/15801412/104184971-f91a4d00-541c-11eb-9074-d57a25e6d8b5.png)
if we merge
![image](https://user-images.githubusercontent.com/15801412/104185041-14855800-541d-11eb-991c-9e622688c9bf.png)

Reply:
Sorry, I still don't understand Vue much and misunderstood what the code was doing, it's fine as it is then! Thanks for the screenshots.